packages/web3-hooks/base/src/useBalance.ts

```diff
@@ -8,8 +8,13 @@ import { useChainContext } from './useContext.js'
 import { useWeb3State } from './useWeb3State.js'
 import { useWeb3Connection } from './useWeb3Connection.js'
 
-export function useBalance<T extends NetworkPluginID = NetworkPluginID>(pluginID?: T, options?: ConnectionOptions<T>) {
-    const { account, chainId } = useChainContext({ account: options?.account })
+export function useBalance<T extends NetworkPluginID = NetworkPluginID>(
+    pluginID?: T,
+    options?: ConnectionOptions<T>,
+    /** Allow to control the request */
+    enabled = true,
```

Comment:
queryOptions

Reply:
Exporting this critical option is enough. Using an option needs to handle merging with the one inside, makes it complicate.